luaui/barwidgets.lua

```diff
@@ -812,6 +811,18 @@ function widgetHandler:InsertWidget(widget)
 		end
 	end
 	self:UpdateCallIns()
+	ki.active = true
```

Comment:
Oops, I actually meant to comment on this `ki.active = true` (instead of https://github.com/beyond-all-reason/Beyond-All-Reason/pull/2546#discussion_r1518118380).

---

Is there a case where it's not already active?

Reply:
it does not look like there is ever a case where `active == false` by that point